context/app/static/js/components/detailPage/DetailPageSection/CollapsibleDetailPageSection.tsx

```diff
@@ -1,14 +1,19 @@
-import React, { PropsWithChildren } from 'react';
+import React, { PropsWithChildren, useState } from 'react';
 import AccordionDetails from '@mui/material/AccordionDetails';
 import AccordionSummary from '@mui/material/AccordionSummary';
 import ExpandMore from '@mui/icons-material/ExpandMore';
 import Typography, { TypographyProps } from '@mui/material/Typography';
 import Box from '@mui/material/Box';
 import { SvgIconComponent } from '@mui/icons-material';
+import useEventCallback from '@mui/material/utils/useEventCallback';
```

Comment:
[MUI v7 removes 4-level deep imports](https://mui.com/material-ui/migration/upgrade-to-v7/#package-layout-updated), so I think we can do this for futureproofing: 

```suggestion
import { useEventCallback }from '@mui/material/utils';
```

Reply:
Nice catch!